build.sbt

```diff
@@ -247,6 +247,7 @@ lazy val `almond-spark` = project
     mimaExceptIn("2.13"),
     libraryDependencies ++= Seq(
       Deps.ammoniteReplApi.value % "provided",
+      Deps.fs2,
```

Comment:
I'm wary of adding this dependency. fs2 is not a dependency of the user-facing API of almond (`cs resolve -r jitpack sh.almond:::scala-kernel-api:0.10.8`). I think it's possible to achieve what's done with fs2 below with the Java standard library.